.github/workflows/virtualbox_image.yml

```diff
@@ -16,6 +16,10 @@ env:
 jobs:
   pyne_image_build:
     runs-on: ubuntu-latest
+    strategy:
+      matrix: 
+        pkg_mgr: ['apt', 'conda']
```

Comment:
I don't think we need both options here.  For users relying on the VM, we can pick only one option - `apt` is probably fine.